src/sentry/tasks/statistical_detectors.py

```diff
@@ -233,6 +233,9 @@ def _detect_transaction_change_points(
             # which was originally intended to detect a gradual regression
             # for the trends use case. That does not apply here.
             "allow_midpoint": "0",
+            "trend_percentage()": 0.5,
```

Comment:
Unsure if this is too high or not, but lets start with this

Reply:
yeah I think it's a good start. my bet is we might want to go higher